### ISSUE TITLE
[Vite] Couldn't resolve host name (Could not resolve host: sentry.io)

### ISSUE THREAD

**Issue by yun548** [getsentry/sentry-cli#2125]
### CLI Version

2.33.1

### Operating System and Architecture

- [ ] macOS (arm64)
- [ ] macOS (x86_64)
- [ ] Linux (i686)
- [x] Linux (x86_64)
- [ ] Linux (armv7)
- [ ] Linux (aarch64)
- [ ] Windows (i686)
- [ ] Windows (x86_64)

### Operating System Version

Ubuntu

### Exact Reproduction Steps

- Execute nuxt build with Vite on a github runner using Vite sentry plugin
- The build fails 

### Expected Results

The build should not fail as the sources have been uploaded 

### Actual Results

The build failed

### Logs

$ nuxt build
Nuxt 3.8.2 with Nitro 2.8.1
ℹ Building client...
ℹ vite v4.5.1 building for production...
[sentry-vite-plugin] Info: Sending telemetry data on issues and performance to Sentry. To disable telemetry, set `options.telemetry` to `false`.
ℹ transforming...

 WARN  [plugin:vite:resolve] Module "crypto" has been externalized for browser compatibility, imported by "/home/elbisna/search/source/services/hasher.ts". See http://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility for more details.

services/hasher.ts (5:4) "createHash" is not exported by "__vite-browser-external", imported by "services/hasher.ts".
ℹ ✓ 655 modules transformed.
ℹ rendering chunks...
ℹ computing gzip size...
ℹ .nuxt/dist/client/_nuxt/assets/plus.14082024a.svg                     0.24 kB │ gzip:   0.20 kB
ℹ .nuxt/dist/client/_nuxt/assets/cross-dark.14082024a.svg               0.33 kB │ gzip:   0.22 kB
ℹ .nuxt/dist/client/_nuxt/assets/cross.14082024a.svg                    0.33 kB │ gzip:   0.23 kB
ℹ .nuxt/dist/client/_nuxt/assets/facebook.14082024a.svg                 0.33 kB │ gzip:   0.24 kB
ℹ .nuxt/dist/client/_nuxt/assets/three-dots-vertical.14082024a.svg      0.41 kB │ gzip:   0.27 kB
ℹ .nuxt/dist/client/_nuxt/assets/bell.14082024a.svg                     0.52 kB │ gzip:   0.31 kB
ℹ .nuxt/dist/client/_nuxt/assets/drop-white.14082024a.svg               0.59 kB │ gzip:   0.40 kB
ℹ .nuxt/dist/client/_nuxt/assets/twitter.14082024a.svg                  0.65 kB │ gzip:   0.35 kB
ℹ .nuxt/dist/client/_nuxt/assets/ios.14082024a.svg                      0.79 kB │ gzip:   0.46 kB
ℹ .nuxt/dist/client/_nuxt/assets/bing-maps.14082024a.svg                0.83 kB │ gzip:   0.51 kB
ℹ .nuxt/dist/client/_nuxt/assets/arrow-right.14082024a.svg              0.87 kB │ gzip:   0.48 kB
ℹ .nuxt/dist/client/_nuxt/assets/arrow-left.14082024a.svg               0.91 kB │ gzip:   0.50 kB
ℹ .nuxt/dist/client/_nuxt/assets/news-bordered.14082024a.svg            0.93 kB │ gzip:   0.55 kB
ℹ .nuxt/dist/client/_nuxt/assets/loader-white.14082024a.svg             0.95 kB │ gzip:   0.33 kB
ℹ .nuxt/dist/client/_nuxt/assets/loader-primary.14082024a.svg           0.96 kB │ gzip:   0.33 kB
ℹ .nuxt/dist/client/_nuxt/assets/menu.14082024a.svg                     0.96 kB │ gzip:   0.51 kB
ℹ .nuxt/dist/client/_nuxt/assets/videos-bordered.14082024a.svg          1.00 kB │ gzip:   0.57 kB
ℹ .nuxt/dist/client/_nuxt/assets/maps-bordered.14082024a.svg            1.01 kB │ gzip:   0.59 kB
ℹ .nuxt/dist/client/_nuxt/assets/images-bordered.14082024a.svg          1.01 kB │ gzip:   0.59 kB
ℹ .nuxt/dist/client/_nuxt/assets/search-bordered.14082024a.svg          1.06 kB │ gzip:   0.62 kB
ℹ .nuxt/dist/client/_nuxt/assets/google-maps.14082024a.svg              1.27 kB │ gzip:   0.66 kB
ℹ .nuxt/dist/client/_nuxt/assets/bookingcom.14082024a.svg               1.37 kB │ gzip:   0.73 kB
ℹ .nuxt/dist/client/_nuxt/assets/search-grey.14082024a.svg              1.53 kB │ gzip:   0.82 kB
ℹ .nuxt/dist/client/_nuxt/assets/android.14082024a.svg                  1.70 kB │ gzip:   0.70 kB
ℹ .nuxt/dist/client/_nuxt/assets/user.14082024a.svg                     1.91 kB │ gzip:   1.27 kB
ℹ .nuxt/dist/client/_nuxt/assets/search.14082024a.svg                   2.07 kB │ gzip:   1.07 kB
ℹ .nuxt/dist/client/_nuxt/assets/instagram.14082024a.svg                2.30 kB │ gzip:   0.88 kB
ℹ .nuxt/dist/client/_nuxt/assets/earth.14082024a.svg                    2.55 kB │ gzip:   1.32 kB
ℹ .nuxt/dist/client/_nuxt/assets/icon-arrow-green.14082024a.png         7.83 kB
ℹ .nuxt/dist/client/manifest.json                                       7.85 kB │ gzip:   0.93 kB
ℹ .nuxt/dist/client/_nuxt/assets/placeholder-1by1.14082024a.svg        12.69 kB │ gzip:   9.30 kB
ℹ .nuxt/dist/client/_nuxt/assets/user.14082024a.png                    19.58 kB
ℹ .nuxt/dist/client/_nuxt/assets/error-astronaut.14082024a.svg         25.51 kB │ gzip:   5.23 kB

 WARN 
(!) Some chunks are larger than 500 kBs after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.

ℹ .nuxt/dist/client/_nuxt/assets/placeholder-3by2.14082024a.svg        65.07 kB │ gzip:  48.07 kB
ℹ .nuxt/dist/client/_nuxt/assets/antibot-bg-mobile.14082024a.png       77.66 kB
ℹ .nuxt/dist/client/_nuxt/assets/logo.14082024a.png                    79.04 kB
ℹ .nuxt/dist/client/_nuxt/assets/logo-baseline.14082024a.png          125.35 kB
ℹ .nuxt/dist/client/_nuxt/assets/antibot-bg-tablet.14082024a.png      182.27 kB
ℹ .nuxt/dist/client/_nuxt/assets/antibot-bg-desktop.14082024a.png     205.14 kB
ℹ .nuxt/dist/client/_nuxt/assets/entry.14082024a.css                  683.76 kB │ gzip:  58.88 kB
ℹ .nuxt/dist/client/_nuxt/entry.14082024a.js                        1,164.65 kB │ gzip: 350.89 kB │ map: 4,937.84 kB
> Found 2 files
> Analyzing 2 sources
> Adding source map references
> Bundled 2 files for upload
> Bundle ID: 35939101-20ff-5bde-8c64-ab7e474a8f3f
> Nothing to upload, all files are on the server
> File processing complete
> Organization: myself-4t
> Project: javascript-vue
> Release: 14082024a
> Dist: None
> Upload type: artifact bundle

Source Map Upload Report
  Scripts
    ~/b7ac5f21-53[67](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:68)-4985-9d0b-830320ff7f01-0.js (sourcemap at entry.14082024a.js.map, debug id b7ac5f21-5367-4985-9d0b-830320ff7f01)
  Source Maps
    ~/b7ac5f21-5367-4985-9d0b-830320ff7f01-0.js.map (debug id b7ac5f21-5367-4985-9d0b-830320ff7f01)
[sentry-vite-plugin] Info: Successfully uploaded source maps to Sentry
ℹ ✓ built in 19.38s
✔ Client built in 19391ms
ℹ Building server...
ℹ vite v4.5.1 building SSR bundle for production...
[sentry-vite-plugin] Info: Sending telemetry data on issues and performance to Sentry. To disable telemetry, set `options.telemetry` to `false`.
ℹ transforming...
ℹ ✓ 367 modules transformed.
ℹ rendering chunks...
ℹ .nuxt/dist/server/styles.mjs                                         0.19 kB
ℹ .nuxt/dist/server/_nuxt/assets/entry-styles.14082024a.mjs            3.70 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-1.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-2.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-4.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-5.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-6.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-7.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-8.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-9.mjs.14082024a.js       1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-10.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-11.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-12.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-13.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-14.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-15.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-16.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-17.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-18.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-19.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-20.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-21.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-22.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-23.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-24.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-25.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-26.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-27.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-29.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-30.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-31.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-32.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-33.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-34.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-35.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-36.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-37.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-38.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-39.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-40.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-41.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-42.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-43.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-44.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-45.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-46.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-47.mjs.14082024a.js      1.20 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-28.mjs.14082024a.js      1.21 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/_nuxt/chunks/entry-styles-3.mjs.14082024a.js       1.21 kB │ map:   0.15 kB
ℹ .nuxt/dist/server/server.mjs                                     1,712.10 kB │ map: 831.42 kB
> Found 96 files
> Analyzing 96 sources
> Adding source map references
> Bundled 96 files for upload
> Bundle ID: a3519ea2-13c5-5bf3-8543-a86943c604f0
> Uploaded files to Sentry
> File upload complete (processing pending on server)
> Organization: myself-4t
> Project: javascript-vue
> Release: 14082024a
> Dist: None
> Upload type: artifact bundle

Source Map Upload Report
  Scripts
    ~/0196698c-9927-43a0-b4f0-99b87aa0404c-2.js (sourcemap at entry-styles-10.mjs.14082024a.js.map, debug id 0196698c-9927-43a0-b4f0-99b87aa0404c)
    ~/05bfbcb9-5198-4a29-9886-089955c79162-13.js (sourcemap at entry-styles-20.mjs.14082024a.js.map, debug id 05bfbcb9-5198-4a29-9886-089955c79162)
    ~/0749343f-fb48-44e9-9437-bd7d7f3f9d15-32.js (sourcemap at entry-styles-38.mjs.14082024a.js.map, debug id 0749343f-fb48-44e9-9437-bd7d7f3f9d15)
    ~/0b2cf6cf-a9da-4df1-b83f-b63ed11022b1-21.js (sourcemap at entry-styles-28.mjs.14082024a.js.map, debug id 0b2cf6cf-a9da-4df1-b83f-b63ed11022b1)
    ~/0e71c876-e43e-41fc-bad9-7cc9ca2ede0c-25.js (sourcemap at entry-styles-31.mjs.14082024a.js.map, debug id 0e71c876-e43e-41fc-bad9-7cc9ca2ede0c)
    ~/0ea48c15-e522-459c-aa95-9b[68](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:69)ecec299a-33.js (sourcemap at entry-styles-39.mjs.14082024a.js.map, debug id 0ea48c15-e522-459c-aa95-9b68ecec299a)
    ~/192d4ec6-2e2b-433f-a5b0-4b3fb4c7c535-18.js (sourcemap at entry-styles-25.mjs.14082024a.js.map, debug id 192d4ec6-2e2b-433f-a5b0-4b3fb4c7c535)
    ~/235346ce-753b-4ade-aceb-19175046b451-47.js (sourcemap at entry-styles-9.mjs.14082024a.js.map, debug id 235346ce-753b-4ade-aceb-19175046b451)
    ~/241d1be9-77a0-4dc2-a365-a15d4265157a-24.js (sourcemap at entry-styles-30.mjs.14082024a.js.map, debug id 241d1be9-77a0-4dc2-a365-a15d4265157a)
    ~/2ef2138b-21ac-4d6d-9932-f6b66d43cdd5-41.js (sourcemap at entry-styles-46.mjs.14082024a.js.map, debug id 2ef2138b-21ac-4d6d-9932-f6b66d43cdd5)
    ~/38d364b8-[69](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:70)0d-4197-97e4-ad6b09a73598-9.js (sourcemap at entry-styles-17.mjs.14082024a.js.map, debug id 38d364b8-690d-4197-97e4-ad6b09a73598)
    ~/3c034acf-1cf4-47e0-968c-30f4b563bb09-34.js (sourcemap at entry-styles-4.mjs.14082024a.js.map, debug id 3c034acf-1cf4-47e0-968c-30f4b563bb09)
    ~/3c380244-7679-4d58-a24f-e63340e572d0-30.js (sourcemap at entry-styles-36.mjs.14082024a.js.map, debug id 3c380244-7679-4d58-a24f-e63340e572d0)
    ~/4112791d-4358-4419-bf5b-891510e806af-8.js (sourcemap at entry-styles-16.mjs.14082024a.js.map, debug id 4112791d-4358-4419-bf5b-891510e806af)
    ~/42193a9e-c218-4532-89f3-75a47e631190-45.js (sourcemap at entry-styles-7.mjs.14082024a.js.map, debug id 42193a9e-c218-4532-89f3-75a47e631190)
    ~/4a101984-3d83-4092-9857-15cf6c65cff3-11.js (sourcemap at entry-styles-19.mjs.14082024a.js.map, debug id 4a101984-3d83-4092-9857-15cf6c65cff3)
    ~/4b1bd455-24ae-4ae0-aeec-10f10d61f417-6.js (sourcemap at entry-styles-14.mjs.14082024a.js.map, debug id 4b1bd455-24ae-4ae0-aeec-10f10d61f417)
    ~/4cb2dc65-f534-48c7-9136-3f7d3dbb1a23-31.js (sourcemap at entry-styles-37.mjs.14082024a.js.map, debug id 4cb2dc65-f534-48c7-9136-3f7d3dbb1a23)
    ~/5051c844-9ed6-4051-85ee-41c192d1bf15-43.js (sourcemap at entry-styles-5.mjs.14082024a.js.map, debug id 5051c844-9ed6-4051-85ee-41c192d1bf15)
    ~/585ff318-8832-4c83-a07f-8726fa147230-10.js (sourcemap at entry-styles-18.mjs.14082024a.js.map, debug id 585ff318-8832-4c83-a07f-8726fa147230)
    ~/5e761d8f-9894-4f3b-8d9b-00284fdbd80b-20.js (sourcemap at entry-styles-27.mjs.14082024a.js.map, debug id 5e761d8f-9894-4f3b-8d9b-00284fdbd80b)
    ~/660b7d78-a0b3-449b-ae77-1a276ae53a55-15.js (sourcemap at entry-styles-22.mjs.14082024a.js.map, debug id 660b7d78-a0b3-449b-ae77-1a276ae53a55)
    ~/665e91ac-ddfb-4c21-9fe4-bf40a62a4d92-37.js (sourcemap at entry-styles-42.mjs.14082024a.js.map, debug id 665e91ac-ddfb-4c21-9fe4-bf40a62a4d92)
    ~/695266c2-5588-4895-a9a4-95d83f6b55bd-1.js (sourcemap at entry-styles-1.mjs.14082024a.js.map, debug id 695266c2-5588-4895-a9a4-95d83f6b55bd)
    ~/7a0488ad-61e4-4725-8182-f61d237293a5-46.js (sourcemap at entry-styles-8.mjs.14082024a.js.map, debug id 7a0488ad-61e4-4725-8182-f61d237293a5)
    ~/80ecd9e5-4858-4910-91d1-7b7a0a7133f0-12.js (sourcemap at entry-styles-2.mjs.14082024a.js.map, debug id 80ecd9e5-4858-4910-91d1-7b7a0a7133f0)
    ~/82873476-ebe9-4991-bc62-683f49355c4e-14.js (sourcemap at entry-styles-21.mjs.14082024a.js.map, debug id 82873476-ebe9-4991-bc62-683f49355c4e)
    ~/832621f0-0437-4904-bcc0-4dc66ebcd723-4.js (sourcemap at entry-styles-12.mjs.14082024a.js.map, debug id 832621f0-0437-4904-bcc0-4dc66ebcd723)
    ~/85a60b1e-e6a9-4249-968b-8795ed215ae9-19.js (sourcemap at entry-styles-26.mjs.14082024a.js.map, debug id 85a60b1e-e6a9-4249-968b-8795ed215ae9)
    ~/92d83ed9-812e-49a8-a73e-98820cee4abf-44.js (sourcemap at entry-styles-6.mjs.14082024a.js.map, debug id 92d83ed9-812e-49a8-a73e-98820cee4abf)
    ~/92e03e41-7d0e-4c19-947a-af798588e147-39.js (sourcemap at entry-styles-44.mjs.14082024a.js.map, debug id 92e03e41-7d0e-4c19-947a-af798588e147)
    ~/969af6cf-2d04-4dde-9cf5-[70](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:71)80a71910bb-26.js (sourcemap at entry-styles-32.mjs.14082024a.js.map, debug id 969af6cf-2d04-4dde-9cf5-7080a71910bb)
    ~/b0ba1fee-3f27-4983-929b-60c7b828c096-35.js (sourcemap at entry-styles-40.mjs.14082024a.js.map, debug id b0ba1fee-3f27-4983-929b-60c7b828c096)
    ~/b2633a91-f98b-4edc-99f5-e21bc8ddd86d-3.js (sourcemap at entry-styles-11.mjs.14082024a.js.map, debug id b2633a91-f98b-4edc-99f5-e21bc8ddd86d)
    ~/b5643912-3b4a-45c9-9d47-8827dbdf3159-38.js (sourcemap at entry-styles-43.mjs.14082024a.js.map, debug id b5643912-3b4a-45c9-9d47-8827dbdf3159)
    ~/c0d7a00d-8cb5-4459-ac58-ae2f5b840773-48.js (sourcemap at server.mjs.map, debug id c0d7a00d-8cb5-4459-ac58-ae2f5b840773)
    ~/c66d627b-d9b0-4376-8c8e-c9[71](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:72)16a87430-5.js (sourcemap at entry-styles-13.mjs.14082024a.js.map, debug id c66d627b-d9b0-4376-8c8e-c97116a87430)
    ~/cca765d0-ceb6-4ec9-bb9d-30f3f71822d6-7.js (sourcemap at entry-styles-15.mjs.14082024a.js.map, debug id cca765d0-ceb6-4ec9-bb9d-30f3f71822d6)
    ~/d6ab1a17-a418-473d-84ed-593d96ac2e29-23.js (sourcemap at entry-styles-3.mjs.14082024a.js.map, debug id d6ab1a17-a418-473d-84ed-593d96ac2e29)
    ~/d9d78c15-e19e-46a8-afe8-465c3b2305d0-16.js (sourcemap at entry-styles-23.mjs.14082024a.js.map, debug id d9d78c15-e19e-46a8-afe8-465c3b2305d0)
    ~/e1a28e61-4ae7-4e76-afd6-a6e6e2b8c8d8-22.js (sourcemap at entry-styles-29.mjs.14082024a.js.map, debug id e1a28e61-4ae7-4e76-afd6-a6e6e2b8c8d8)
    ~/e22e134e-9d43-4868-9028-64a0a98ce3e4-17.js (sourcemap at entry-styles-24.mjs.14082024a.js.map, debug id e22e134e-9d43-4868-9028-64a0a98ce3e4)
    ~/e40afc30-887f-46ef-bc22-80d323cc5be8-42.js (sourcemap at entry-styles-47.mjs.14082024a.js.map, debug id e40afc30-887f-46ef-bc22-80d323cc5be8)
    ~/e4cf5702-abe0-4aa2-ad20-570d5b7a9904-36.js (sourcemap at entry-styles-41.mjs.14082024a.js.map, debug id e4cf5702-abe0-4aa2-ad20-570d5b7a9904)
    ~/e97e2bc4-b076-4d8a-9b3a-66afdf42fa7a-27.js (sourcemap at entry-styles-33.mjs.14082024a.js.map, debug id e97e2bc4-b076-4d8a-9b3a-66afdf42fa7a)
    ~/f66faf9e-e203-4452-810e-01d5614294a7-40.js (sourcemap at entry-styles-45.mjs.14082024a.js.map, debug id f66faf9e-e203-4452-810e-01d5614294a7)
    ~/f987e559-13ff-416f-9fa9-1415be3cfa86-29.js (sourcemap at entry-styles-35.mjs.14082024a.js.map, debug id f987e559-13ff-416f-9fa9-1415be3cfa86)
    ~/fca52eae-b2fa-4b4a-a982-0209bdeab9bb-28.js (sourcemap at entry-styles-34.mjs.14082024a.js.map, debug id fca52eae-b2fa-4b4a-a982-0209bdeab9bb)
  Source Maps
    ~/0196698c-9927-43a0-b4f0-99b87aa0404c-2.js.map (debug id 0196698c-9927-43a0-b4f0-99b87aa0404c)
    ~/05bfbcb9-5198-4a29-9886-089955c79162-13.js.map (debug id 05bfbcb9-5198-4a29-9886-089955c79162)
    ~/0749343f-fb48-44e9-9437-bd7d7f3f9d15-32.js.map (debug id 0749343f-fb48-44e9-9437-bd7d7f3f9d15)
    ~/0b2cf6cf-a9da-4df1-b83f-b63ed11022b1-21.js.map (debug id 0b2cf6cf-a9da-4df1-b83f-b63ed11022b1)
    ~/0e71c876-e43e-41fc-bad9-7cc9ca2ede0c-25.js.map (debug id 0e71c876-e43e-41fc-bad9-7cc9ca2ede0c)
    ~/0ea48c15-e522-459c-aa95-9b68ecec299a-33.js.map (debug id 0ea48c15-e522-459c-aa95-9b68ecec299a)
    ~/192d4ec6-2e2b-433f-a5b0-4b3fb4c7c535-18.js.map (debug id 192d4ec6-2e2b-433f-a5b0-4b3fb4c7c535)
    ~/235346ce-753b-4ade-aceb-19175046b451-47.js.map (debug id 235346ce-753b-4ade-aceb-19175046b451)
    ~/241d1be9-77a0-4dc2-a365-a15d4265157a-24.js.map (debug id 241d1be9-77a0-4dc2-a365-a15d4265157a)
    ~/2ef2138b-21ac-4d6d-9932-f6b66d43cdd5-41.js.map (debug id 2ef2138b-21ac-4d6d-9932-f6b66d43cdd5)
    ~/38d364b8-690d-4197-97e4-ad6b09a73598-9.js.map (debug id 38d364b8-690d-4197-97e4-ad6b09a73598)
    ~/3c034acf-1cf4-47e0-968c-30f4b563bb09-34.js.map (debug id 3c034acf-1cf4-47e0-968c-30f4b563bb09)
    ~/3c380244-7679-4d58-a24f-e63340e5[72](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:73)d0-30.js.map (debug id 3c380244-7679-4d58-a24f-e63340e572d0)
    ~/4112791d-4358-4419-bf5b-891510e806af-8.js.map (debug id 4112791d-4358-4419-bf5b-891510e806af)
    ~/42193a9e-c218-4532-89f3-75a47e631190-45.js.map (debug id 42193a9e-c218-4532-89f3-75a47e631190)
    ~/4a101984-3d83-4092-9857-15cf6c65cff3-11.js.map (debug id 4a101984-3d83-4092-9857-15cf6c65cff3)
    ~/4b1bd455-24ae-4ae0-aeec-10f10d61f417-6.js.map (debug id 4b1bd455-24ae-4ae0-aeec-10f10d61f417)
    ~/4cb2dc65-f534-48c7-9136-3f7d3dbb1a23-31.js.map (debug id 4cb2dc65-f534-48c7-9136-3f7d3dbb1a23)
    ~/5051c844-9ed6-4051-85ee-41c192d1bf15-43.js.map (debug id 5051c844-9ed6-4051-85ee-41c192d1bf15)
    ~/585ff318-8832-4c83-a07f-8726fa147230-10.js.map (debug id 585ff318-8832-4c83-a07f-8726fa147230)
    ~/5e761d8f-9894-4f3b-8d9b-00284fdbd80b-20.js.map (debug id 5e761d8f-9894-4f3b-8d9b-00284fdbd80b)
    ~/660b7d78-a0b3-449b-ae77-1a276ae53a55-15.js.map (debug id 660b7d78-a0b3-449b-ae77-1a276ae53a55)
    ~/665e91ac-ddfb-4c21-9fe4-bf40a62a4d92-37.js.map (debug id 665e91ac-ddfb-4c21-9fe4-bf40a62a4d92)
    ~/695266c2-5588-4895-a9a4-95d83f6b55bd-1.js.map (debug id 695266c2-5588-4895-a9a4-95d83f6b55bd)
    ~/7a0488ad-61e4-4725-8182-f61d237293a5-46.js.map (debug id 7a0488ad-61e4-4725-8182-f61d237293a5)
    ~/80ecd9e5-4858-4910-91d1-7b7a0a7133f0-12.js.map (debug id 80ecd9e5-4858-4910-91d1-7b7a0a7133f0)
    ~/828[73](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:74)476-ebe9-4991-bc62-683f49355c4e-14.js.map (debug id 82873476-ebe9-4991-bc62-683f49355c4e)
    ~/832621f0-0437-4904-bcc0-4dc66ebcd723-4.js.map (debug id 832621f0-0437-4904-bcc0-4dc66ebcd723)
    ~/85a60b1e-e6a9-4249-968b-8795ed215ae9-19.js.map (debug id 85a60b1e-e6a9-4249-968b-8795ed215ae9)
    ~/92d83ed9-812e-49a8-a73e-98820cee4abf-44.js.map (debug id 92d83ed9-812e-49a8-a73e-98820cee4abf)
    ~/92e03e41-7d0e-4c19-947a-af798588e147-39.js.map (debug id 92e03e41-7d0e-4c19-947a-af798588e147)
    ~/969af6cf-2d04-4dde-9cf5-7080a71910bb-26.js.map (debug id 969af6cf-2d04-4dde-9cf5-7080a71910bb)
    ~/b0ba1fee-3f27-4983-929b-60c7b828c096-35.js.map (debug id b0ba1fee-3f27-4983-929b-60c7b828c096)
    ~/b2633a91-f98b-4edc-99f5-e21bc8ddd86d-3.js.map (debug id b2633a91-f98b-4edc-99f5-e21bc8ddd86d)
    ~/b5643912-3b4a-45c9-9d47-8827dbdf3159-38.js.map (debug id b5643912-3b4a-45c9-9d47-8827dbdf3159)
    ~/c0d7a00d-8cb5-4459-ac58-ae2f5b840773-48.js.map (debug id c0d7a00d-8cb5-4459-ac58-ae2f5b840773)
    ~/c66d627b-d9b0-4376-8c8e-c97116a8[74](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:75)30-5.js.map (debug id c66d627b-d9b0-4376-8c8e-c97116a87430)
    ~/cca765d0-ceb6-4ec9-bb9d-30f3f71822d6-7.js.map (debug id cca[76](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:77)5d0-ceb6-4ec9-bb9d-30f3f71822d6)
    ~/d6ab1a17-a418-473d-84ed-593d96ac2e29-23.js.map (debug id d6ab1a17-a418-473d-84ed-593d96ac2e29)
    ~/d9d[78](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:79)c15-e19e-46a8-afe8-465c3b2305d0-16.js.map (debug id d9d78c15-e19e-46a8-afe8-465c3b2305d0)
    ~/e1a28e61-4ae7-4e76-afd6-a6e6e2b8c8d8-22.js.map (debug id e1a28e61-4ae7-4e76-afd6-a6e6e2b8c8d8)
    ~/e22e134e-9d43-4868-9028-64a0a98ce3e4-17.js.map (debug id e22e134e-9d43-4868-9028-64a0a98ce3e4)
    ~/e40afc30-887f-46ef-bc22-[80](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:81)d323cc5be8-42.js.map (debug id e40afc30-887f-46ef-bc22-80d323cc5be8)
    ~/e4cf5702-abe0-4aa2-ad20-570d5b7a9904-36.js.map (debug id e4cf5702-abe0-4aa2-ad20-570d5b7a9904)
    ~/e97e2bc4-b076-4d8a-9b3a-66afdf42fa7a-27.js.map (debug id e97e2bc4-b076-4d8a-9b3a-66afdf42fa7a)
    ~/f66faf9e-e203-4452-[81](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:82)0e-01d5614294a7-40.js.map (debug id f66faf9e-e203-4452-810e-01d5614294a7)
    ~/f987e559-13ff-416f-9fa9-1415be3cfa86-29.js.map (debug id f987e559-13ff-416f-9fa9-1415be3cfa86)
    ~/fca52eae-b2fa-4b4a-a9[82](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:83)-0209bdeab9bb-28.js.map (debug id fca52eae-b2fa-4b4a-a982-0209bdeab9bb)

 ERROR  [sentry-debug-id-upload-plugin] Command failed: /home/elbisna/search/source/node_modules/@sentry/cli-linux-x64/bin/sentry-cli releases finalize 14082024a
error: API request failed
  caused by: [6] Couldn't resolve host name (Could not resolve host: sentry.io)

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.

 ERROR  Nuxt Build Error: Command failed: /home/elbisna/search/source/node_modules/@sentry/cli-linux-x64/bin/sentry-cli releases finalize 14082024a
error: API request failed
  caused by: [6] Couldn't resolve host name (Could not resolve host: sentry.io)

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.

  error: API request failed
  caused by: [6] Couldn't resolve host name (Could not resolve host: sentry.io)
  
  Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
  Please attach the full debug log to all bug reports.
  
  at ChildProcess.exithandler (node:child_process:419:12)
  at ChildProcess.emit (node:events:513:28)
  at maybeClose (node:internal/child_process:10[91](https://github.com/liloteam/nuxt-search/actions/runs/10383958476/job/28750129449#step:5:92):16)
  at Socket.<anonymous> (node:internal/child_process:449:11)
  at Socket.emit (node:events:513:28)
  at Pipe.<anonymous> (node:net:320:12)

[sentry-vite-plugin] Info: Successfully uploaded source maps to Sentry
ℹ ✓ built in 21.00s
error Command failed with exit code 1.

**Comment by szokeasaurusrex** [comment]
@yun548 It looks like this issue might be a duplicate of #1929. Could you check whether the setup and problem in that issue matches your setup? If your setup or problem is different, please explain how.

**Comment by yun548** [comment]
@szokeasaurusrex I don't believe the issue is contained to rootless docker as we have encountered that error on two other environments  : 
- MacOs with apple CPU running nodejs installed with brew (so no docker)
- Github actions running the following workflow : 
```
name: test_and_deploy

on:
  push:

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'
        required: true
        default: 'warning'
        type: choice
        options:
          - info
          - warning
          - debug

jobs:
  test:
    runs-on: ${{ matrix.os }}

    strategy:
      matrix:
        os: [ubuntu-latest]
        node: [18.18.2]

    steps:
      - name: Checkout 🛎
        uses: actions/checkout@master

      # Does some stuff

  deploy_prod_v2:
    needs: test
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/prod_v2'

    steps:
      - name: Checkout 🛎
        uses: actions/checkout@v2

      - name: Setup SSH 🏗
        uses: MrSquaare/ssh-setup-action@v2
        with:
          host: github.com
          private-key: ${{ secrets.PRIVATE_SSH_KEY }}

      - name: Install PM2 ‍💻
        run: npm install pm2 -g

      # Build fails here
      - name: Deploy 🚀
        run: pm2 deploy prod_v2 --env prod

```

**Comment by szokeasaurusrex** [comment]
Hmm interesting. Has this workflow ever worked for you before, for example, in a previous Sentry CLI version? If yes, when did you notice the issue starting?

**Comment by yun548** [comment]
> Hmm interesting. Has this workflow ever worked for you before, for example, in a previous Sentry CLI version? If yes, when did you notice the issue starting?

Hi, yes the workflow has worked before, the error happens randomly

**Comment by szokeasaurusrex** [comment]
@yun548 got it. Did you notice the issue starting after you upgraded the Sentry CLI to a specific version?

**Comment by yun548** [comment]
@szokeasaurusrex We are using `@sentry/vite-plugin@^2.22.0` we haven't tested with other versions but I do not mind helping you test that if there are specific versions you would like me to try to reproduce one.

**Comment by szokeasaurusrex** [comment]
@yun548 can you check which version of `@sentry/cli` you are running? Perhaps you could find this in your lock file

**Comment by yun548** [comment]
@szokeasaurusrex we are using 2.33.1

**Comment by szokeasaurusrex** [comment]
Hmm, maybe you could try bumping to the latest version (currently 2.36.1)? This also seems like it could just be some sort of network issue; do you ever face similar error messages when using other services on this system
